packages/server/graphql/mutations/addReactjiToReactable.ts

```diff
@@ -69,11 +76,15 @@ const addReactjiToReactable = {
     const subOptions = {mutatorId, operationId}
 
     //AUTH
-    const dbTable = tableLookup[reactableType]
-    const reactable = (await r
-      .table(dbTable)
-      .get(reactableId)
-      .run()) as Reactable
+    let reactable: Reactable
+    const isPgTable = reactableType === 'RESPONSE'
+    const rethinkDbTable = rethinkTableLookup[reactableType]
+    if (isPgTable) {
+      const loaderName = pgDataloaderLookup[reactableType]
+      reactable = (await dataLoader.get(loaderName).load(reactableId)) as Reactable
```

Comment:
+1 `.loadNonNull(reactableId)` and then you can remove the casting

Reply:
+1 clear the dataloader cache right here on L85 so you don't need to do it at the bottom of the function

Reply:
I disagree with clearing the cache here, because although we've performed the only read before the cache is invalidated, we haven't actually done anything to invalidate the cache.  The current placement of the cache-clearing logic is immediately after the underlying data is mutated.

Reply:
I've made this a bit clearer by rearranging the `isPg` and `isRemove` branches such that the cache-clearing happens at the end of the `isPg` branch containing both removals and additions